hardware/Layout-Desing-Guide.md

```diff
@@ -0,0 +1,486 @@
+
+
+# 1. Why is the proper layout design important? 
+
+If the layout is an inappropriate design that can cause 
++ Degradation in the communication distance 
++ The unit can violate the relevant EMC regulation (high harmonics, spur) 
++ The battery life can be reduced
+
+# 2. RF section 
+
+
+## 2.1. Different parts of RF section
+The most important part of the proper layout design is the RF circuitry itself.
+
+In the case of dual band RFIC, this RF part includes the following:
+1. Sub-GHz and 2.4 GHz RF matching newtork  
+2. Power supply filterings 
+3. High frequency crystal  
+
+
+
+<p align="center">
+  "<img src="files/Layout-Design-Guide/matching network.png">
+</p>
+
+<div align="center">
+
+<b>Figure 1. RF section</b> 
+
+</div>
+
+     
+
+
+### 2.1.1. Function of RF matching  and filtering network
+
+The RF transmitter ICs can synthetize high frequency and high power signals at their TX output: the fundamental, its harmonics and additional spurious content.
+On the Silicon Labs’ reference design boards the function of the TX matching is to drive most of the fundamental power into the antenna, thus to provide the required impedance transformation between the RFIC and the load. Besides that, a proper filtering network should be used between the matching network and the antenna design which is able to suppress harmonics and spurs. RX matching network is able to provide impedancia tuning between IC and the load. 
+
+
+> **TX matching:** 
+> + Deliver the required TX RF power
+> + Impedance transformation
+> + Harmonic/spur suppression 
+
+
+
+> **RX matching:**  
+> + Impedance match  
+
+
+### 2.1.2. Function of VDD filterings
+
+Power supplies, high speed blocks and converters mounted on the board can cause extra spurs in the RF spectrum or sensitivity degradation.
+So, the filtering and the isolation of these blocks from the RF part are important. The lowest value capacitors filter out the fundamental frequency. The capacitor with the value around 100nF filters the several tenths of MHz frequency range, thus the leaking clock spurs. Filtering these spurs is important as they can be upconverted inside the chip causing unwanted spurs at several 10MHz offset around the carrier. The largest value capacitor filters the several hundred kHz interferences typically coming from a switching power supply. 
+
+
+
+****
+ 
+## 2.2. Layout of RF section    
+
+Silicon Labs suggests copying the RF part of the reference PCB design, or if it is not possible, applying the layout design rules and guidelines described in [AN928.1](https://www.silabs.com/documents/public/application-notes/an928.1-efr32-series1-layout-design-guide.pdf)
+
+
+<p align="center">
+  <img src="files/Layout-Design-Guide/Rf part.png">
```

Comment:
The name of the image should be "RF part.png". The GitHub is case sensitive not like Windows